code/game/machinery/telecomms/presets.dm

```diff
@@ -51,22 +51,23 @@
 	id = "Receiver A"
 	network = "tcommsat"
 	autolinkers = list("receiverA") // link to relay
-	freq_listening = list(1351, 1355, 1347, 1349) // science, medical, supply, service
 
+/obj/machinery/telecomms/receiver/preset_left/initialize()
+	freq_listening = list(SCI_FREQ, MED_FREQ, SUP_FREQ, SER_FREQ) // science, medical, supply, service
+	..()
 
 //--PRESET RIGHT--//
 
 /obj/machinery/telecomms/receiver/preset_right
 	id = "Receiver B"
 	network = "tcommsat"
 	autolinkers = list("receiverB") // link to relay
-	freq_listening = list(1345, 1353, 1357, 1359) //ert, command, engineering, security
-
-	//Common and other radio frequencies for people to freely use
-	New()
-		for(var/i = 1441, i < 1489, i += 2)
-			freq_listening |= i
-		..()
+	
+/obj/machinery/telecomms/receiver/preset_right/initialize()
```

Comment:
Moved the `new()` code into `initialize()` because I didn't feel there was a particular reason for it to be on `new()`

Reply:
But you're not calling initialize in New if the game already started

Reply:
So you mean this should all be in new() ?

Reply:
I'm saying initialize won't run if you create one of these after round start
if for some reason one was to do that

Reply:
I get it. Since it's the `preset` receiver I'm not sure if people will actually create them.